script/update-action-docs

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Shiuld we add this to precommit eventually?

Reply:
oh, I could just add it locally...  why don't I do that for now.